src/MaterialDesignThemes.Wpf/Converters/NullableToVisibilityConverter.cs

```diff
@@ -5,6 +5,9 @@ namespace MaterialDesignThemes.Wpf.Converters;
 
 public class NullableToVisibilityConverter : IValueConverter
 {
+    public static readonly NullableToVisibilityConverter Instance = new();
+    public static readonly NullableToVisibilityConverter InverseInstance = new() { NullValue = Visibility.Visible, NotNullValue = Visibility.Hidden };
```

Comment:
I think we probably want a different name here since it is not actually the inverse of Instance. Thoughts on `NotHiddenInstance`?

Reply:
This will be changed to the same naming convention that we decide for the BooleanToVisibilityConverter.